templates/base/server/services/auth/__init__.py

```
@@ -0,0 +1,33 @@
+from server.utils.security import get_password_manager
+from server.utils.security import get_token_manager
+from server.utils.security.devices import DeviceManager
+from server.utils.security.password import PasswordManager
+from server.utils.security.tokens import TokenManager
+from sqlmodel.ext.asyncio.session import AsyncSession
+
+from .service import AuthService
+
+
+def get_auth_service(
+    session: AsyncSession = Depends(get_session),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)